File: groups-service/src/main/java/com/ironhack/groupsservice/model/GroupsMembers.java
package com.ironhack.groupsservice.model;

import javax.persistence.*;

@Entity
@Table(name = "groups_members")
public class GroupsMembers {

    /**
    * Properties
     */
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private Long UserId;
    @ManyToOne
    @JoinColumn(name = "group_id")
    private Groups groups;

    /**
     * Default constructor
     */
    public GroupsMembers() {
    }

    /**
     * Constructor: userId, groups
     */
    public GroupsMembers(Long userId, Groups groups) {
        UserId = userId;
        this.groups = groups;
    }

    /**
     * Getters & Setters
     */
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getUserId() {
        return UserId;
    }

    public void setUserId(Long userId) {
        UserId = userId;
    }

    public Groups getGroups() {
        return groups;
    }

    public void setGroups(Groups groups) {
        this.groups = groups;
    }
}

File: grouping-client/src/app/models/groups-members.ts
export class GroupsMembers {

  constructor(
    private _id: number,
    private _userId: number,
    private _groupId: number
  ) {}

  public get id(): number {
    return this._id;
  }
  public set id(value: number) {
    this._id = value;
  }
  public get userId(): number {
    return this._userId;
  }
  public set userId(value: number) {
    this._userId = value;
  }
  public get groupId(): number {
    return this._groupId;
  }
  public set groupId(value: number) {
    this._groupId = value;
  }
}
